docs: update AppKit terminology to Reown across documentation (#738)
reown-com/reown-docs: appkit/react-native/core/options.mdx

### `tokens`
*   **Type:** `Record<string, { address: string }>` (Optional)

> *   **Description:** An object to configure specific tokens for which Reown should display balances. The keys are CAIP-2 network IDs (e.g., 'eip155:1' for Ethereum Mainnet for a specific Solana network instance if required by ID structure). The value for each network ID is an object containing the token's contract `address` on that network.

Reviewer: ```suggestion
*   **Description:** An object to configure specific tokens the SDK will use to display balances. The keys are CAIP-2 network IDs (e.g., 'eip155:1' for Ethereum Mainnet for a specific Solana network instance if required by ID structure). The value for each network ID is an object containing the token's contract `address` on that network.
```
